src/MICore/LaunchOptions.cs

```diff
@@ -457,7 +455,7 @@ static internal LocalLaunchOptions CreateFromJson(JObject parsedOptions)
 
             if (launchOptions == null)
             {
-                throw new InvalidLaunchOptionsException(MICoreResources.Error_InvalidLaunchOptions);
+                throw new InvalidLaunchOptionsException(MICoreResources.Error_LaunchOptionsNull);
```

Comment:
>Error_LaunchOptionsNull [](start = 72, length = 23)

Is this just an internal error? Or can the user actually do something to get here? If it is just an internal error, we might not want to add a new localizable string. If it is not just an internal error, should we better describe what is wrong with the launch options?